### PR TITLE
feat(example data): add a command to load example data

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
Nice!

`initial_data.json` only needs to contain records for the models of our own app (`alexandria_core`). So you can remove everything except the categories at the bottom.

Edit: Also a small test would be nice to confirm, that loading the data actually works.

**Comment by velrest** [comment]
@open-dynaMIX added a tiny test

File: alexandria/core/tests/test_example_data.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+def test_load_example_data(db):
+
+    assert len(Category.objects.all()) == 0
+
+    call_command("loaddata", "initial_data.json")
+
+    assert len(Category.objects.all()) == 5
+    assert Category.objects.first().color == "#CB68C1"
```
Comment (open-dynaMIX) [inline alexandria/core/tests/test_example_data.py:15]:
```suggestion
    assert Category.objects.get(pk="alle-beteiligten").color == "#CB68C1"
```

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
Awesome!

Just a small nitpick.

File: initial_data.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            "created_by_user": null,
+            "created_by_group": null,
+            "modified_at": "2020-08-04T13:24:00.704Z",
+            "modified_by_user": null,
+            "modified_by_group": null,
+            "meta": {
+
+            },
+            "name": "{\"en\": \"Alle Beteiligten\", \"de\": \"Alle Beteiligten\"}",
+            "description": "{\"en\": \"\"}",
```
Comment (open-dynaMIX) [inline initial_data.json:16]: As this is our demo data, please be consistent with translations. Most objects here have a name in german and a description in english.

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]

File: initial_data.json (hunk 2 above)
Comment (velrest) [reply initial_data.json:16]: I've added Englisch names everywhere

**Review by open-dynaMIX — APPROVED** [review]

**Pull request merged**